app/Http/Controllers/ItemController.php

```diff
@@ -77,7 +77,11 @@ public function update(ItemUpdateRequest $request, Item $item)
 
         $request->session()->flash('item.id', $item->id);
 
-        return redirect()->route('item.index');
+        if ($charId = $item->character_id) {
```

Comment:
just so i'm sure i understand, this is checking if $item->character_id exists or rather isn't null?

Reply:
@IanjhPhillips That's exactly right, at the same time it sets a variable `$charId`; in loosely typed langauges, usually null is evaluated as a "falsey" value :)